### PR TITLE
2019 application

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

File: application.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Has your org been accepted as a mentor org in GSoC before?
+
+Yes.
+
+## Which years did you participate in GSoC?
+
+2014, 2015, 2016, 2018.
+
+## For each year your org has participated, provide the counts of successful and total students:
```
Comment (segiddins) [inline application.md:47]: need to get these numbers

File: application.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+2014:
+2015:
+2016:
+2018:
+
+## If your org has applied for GSoC before but not been accepted, select the years:
+
+None
```
Comment (segiddins) [inline application.md:55]: is this correct?

File: application.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Provide a reference: (optional)
+
+Intentionally left blank since we're not a first-time org
```
Comment (segiddins) [inline application.md:59]: Unless there's a ruby contact we have at Google we'd like to provide?

File: application.md (hunk 2 above)
Comment (indirect) [reply application.md:55]: yup, we have never been rejected. only Rails has been rejected some years.

**Comment by segiddins** [comment]
What's on here now is on our application, so you have ~a day left to provide feedback.

**Review by indirect — APPROVED** [review]
👍

**Pull request merged**